src/ape_ethereum/ecosystem.py

```diff
@@ -490,3 +490,37 @@ def decode_value(t, v) -> Any:
                 block_hash=log["blockHash"],
                 block_number=log["blockNumber"],
             )  # type: ignore
+
+    def decode_ds_note(self, log: dict) -> Optional[ContractLog]:
```

Comment:
Is this just a useful helper?

Reply:
yes, it's used to decode [custom logs](https://github.com/dapphub/ds-note) which encode the function call and args. these are common in makerdao contracts.

Reply:
We could call it `decode_dappsys_note()` to give more context (optional)

Reply:
this library is more known as ds-note